src/useKeyboardEvent/useKeyboardEvent.stories.mdx

```diff
@@ -46,27 +46,32 @@ function useKeyboardEvent(
 ## Usage
 
 ```ts
-useKeyboardEvent(Key.ArrowRight, (event: KeyboardEvent) => {
+useKeyboardEvent('ArrowRight', (event: KeyboardEvent) => {
   console.log('The arrow right is pressed', event.currentTarget);
 });
 ````
 
 ```ts
+import { Key } from "ts-key-enum"
+
 const Demo = defineComponent({
   name: 'demo',
   refs: {
     someRef: 'some-ref'
   },
   setup({ refs }) {
 
-    // The default event is bound to the document
-    useKeyboardEvent(Key.ArrowRight, () => console.log('The `ArrowRight` key is pressed.'));
+    // The default event is bound to the document and you can use
+    useKeyboardEvent('a', () => console.log('The `a` key is pressed.'));
+
+    // You can also use the `ts-key-enum` to reference keys
+    useKeyboardEvent(Key.ArrowLeft, () => console.log('The `ArrowLeft` key is pressed.'));
 
     // You can add multiple keys to your listener, either through the `ts-key-enum` or a `string`
-    useKeyboardEvent([Key.ArrowLeft, 'a'], () => console.log('The `ArrowLeft` or `a` key was pressed.'));
+    useKeyboardEvent([Key.ArrowUp, Key.ArrowDown], () => console.log('The `ArrowLeft` or `ArrowDown` key was pressed.'));
```

Comment:
I think you intended to use the `'a'` string there, since the comment says:
> either through the `ts-key-enum` **or a `string`**

Reply:
Yeah, I moved the use case for the `a` key to a standalone example since that is the base, the other ones just use `ts-key-enum`